src/components/dashboard/admin/upload-votes.tsx

```diff
@@ -117,13 +118,13 @@ export const UploadVotes: React.FunctionComponent = () => {
     const convertFileToVotesArray = (voteBatch: VoteBatchFile): Array<Vote> => {
         const votesArray: Array<string> = voteBatch.text.split('\n');
         const votes: Array<Vote> = votesArray.map((item) => {
+            const values = item.split(',');
```

Comment:
Should you not be splitting by tabs (\t) rather than comma (,)?

Reply:
if we are uploading a tab separated file then yeah that would make sense, the testing I did was with a comma separated one